nebu/tests/cli/test_get.py

```diff
@@ -83,6 +83,62 @@ def _rel(p, b):
                                 pathlib_walk(expected))
         assert sorted(relative_dir) == sorted(relative_expected)
 
+    def test_three_part_vers(self, datadir, tmpcwd, monkeypatch, requests_mock,
```

Comment:
What this test is doing is: 
- when user requests a specific version (in this case `1.1.1`) other than and older than current "latest" (in this case `1.1.4`)
- expect version `1.1.4` to be dowloaded.
Is this correct?

What happens if user requests a version **newer** than latest? Could you add a test for that?